### PR TITLE
Don't clone history

### PR DESCRIPTION
**Pull request by tim-gromeyer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ cd src
 lrelease QOwnNotes.pro
 
 qmake
-make
+make -j4
```
Comment (pbek) [inline README.md:160]: That depends on the amount of cores the system has...

Comment (tim-gromeyer) [reply README.md:160]: Yeah, but [here](https://github.com/pbek/QOwnNotes#building-qownnotes-for-development) it's the same

Comment (pbek) [reply README.md:160]: Ok, then :)

**Comment by pbek** [comment]
Thank you

**Pull request merged**